test.md

```diff
@@ -47,4 +47,6 @@
 - A) K zaznamenání všech commitů.
 - B) K definování pravidel pro ignorování souborů.
 - C) K definování vzdálených repozitářů.
-- D) K zobrazení aktuálního stavu repozitáře.
\ No newline at end of file
+- D) K zobrazení aktuálního stavu repozitáře.
+
```

Comment:
Super, i více prádzných řádků je vlastně změna a hodně velký hnidpich by mohl chtít po tobě vrátit soubor do opravdu původní podoby tak, aby tam nebyla žádná změna zaznamenaná v gitu, ale my si na to dneska nehrajeme :)